### PR TITLE
Adding note about badge requirements, per Brienne.

### PR DESCRIPTION
**Pull request by jberkus**
/assign @salaxander

### CODE REVIEW

**Comment by tokt** [comment]
Thank you! We should probably mention (a URL?) when/where they'll be able to get the badge.
(Probably closer to the event when more details are known).

**Comment by reylejano** [comment]
> Thank you! We should probably mention (a URL?) when/where they'll be able to get the badge. (Probably closer to the event when more details are known).

Badge pickup is on sched, https://kccncna2024.sched.com/
The two Monday sched entries for badge locations are:
https://kccncna2024.sched.com/event/1iGUa/badge-pick-up
https://kccncna2024.sched.com/event/1kiw6/badge-pick-up

Folks should have their badges to attend KCS before the social but travel delays are a possibility

File: content/en/events/2024/kcsna/social.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ There will be food, live music, activities, and more!
 
 *The contributor social is _not_ a public event.  You must be [registered] in advance to attend the Contributor Summit in order to attend the Social, and must be a Kubernetes contributor.  Community members who are not registered and show up at the Social will be turned away.*
 
+**You must bring your Kubecon badge to get into the Social.**
```
Comment (stmcginnis) [inline content/en/events/2024/kcsna/social.md:12]: Extreme nit: KubeCon

Feel free to ignore, but I have a compulsion to point it out. :D

**Review by stmcginnis — COMMENTED** [review]
/lgtm

**Pull request merged**